X-CleverCloud-Header header to identify a monitoring ping (#469)
CleverCloud/documentation: content/doc/find-help/faq.md

To create add-ons, you need to complete your account information, including your city and ZIP code. 
For instance, you cannot create a Matomo add-on until you provide these details.

## I get unknown regular requests, is there a problem ? 

> For monitoring purposes, our system regularly talks to apps in HTTP to ensure they're operationnal. This ping occurs regularly (every 2mn), and might appear in your logs. This is not a safety risk. 

Reviewer: `This is not a safety risk. `
Agree with David, maybe deleting this part is okay. We already say that this request come from our monitoring in the sentence before.
Otherwise `this is a legitimate request from our systems` is also good.

> For monitoring purposes, our system regularly talks to apps in HTTP to ensure they're operationnal. This ping occurs regularly (every 2mn), and might appear in your logs. This is not a safety risk. 

Reviewer 2: ```suggestion
Our system performs routine health checks by sending HTTP requests to applications every 2 minutes. You may notice these periodic pings in your logs - they're simply part of our standard monitoring process.
```

To create add-ons, you need to complete your account information, including your city and ZIP code. 
For instance, you cannot create a Matomo add-on until you provide these details.

## I get unknown regular requests, is there a problem ? 

> The platform performs routine health checks by sending HTTP requests to applications every 2 minutes. You may notice these periodic pings in your logs - they're part of the standard monitoring process.
> You can identify this ping by filtering `X-Clever-Monitoring` requests with this header are legitimate ones from our monitoring system. 

Reviewer 3: ```suggestion
The platform performs routine health checks to applications every 2 minutes. You may notice these periodic HTTP requests in your logs, with `X-Clever-Monitoring` header. They're part of Clever Cloud's standard monitoring process.
```
